docs/skopeo.1.md

```diff
@@ -19,7 +19,7 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
 
   **docker://**_docker-reference_
-  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$HOME/.docker/config.json`, which is set e.g. using `(docker login)`.
+  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in either `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(kpod login)` or in `$HOME/.docker/config.json`, which is set using `(docker login)`
```

Comment:
👍 
@umohnani8 I do think this should probably be added to kpod/buildah somewhere.  We're currently searching thorough containers/auth.json and then .docker/config.json in both of those now right?  If not, then this should be added when we are.

Reply:
I think we probably ought to note precedence here too.  Perhaps:  "By default, uses the authorization state in $XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(kpod login)`.  If the authorization state is not found there, then `$HOME/.docker/config.json` is checked which is set using `(docker login)`."

FWIW, missing a period at the end of your current line here.

Reply:
fixed.